execution/evm/vm_test.go

```diff
@@ -158,6 +158,26 @@ func TestSubcurrency(t *testing.T) {
 	}
 }
 
+func TestRevert(t *testing.T) {
+	ourVm := NewVM(newAppState(), DefaultDynamicMemoryProvider, newParams(), acm.ZeroAddress, nil, logger)
+
+	// Create accounts
+	account1 := newAccount(1)
+	account2 := newAccount(1, 0, 1)
+
+	var gas uint64 = 100000
+
+	bytecode := MustSplice(PUSH32, 0x72, 0x65, 0x76, 0x65, 0x72, 0x74, 0x20, 0x6D, 0x65, 0x73, 0x73, 0x61,
```

Comment:
Cool...  It this bytecode the testcase from the EIP? If so can you add a comment with a link and reference to EIP-140